docs/_guides/codewind-crw-quick-guide.md

```diff
@@ -5,14 +5,13 @@ title: "Getting Started with Codewind in CodeReady Workspaces"
 categories: guides 
 description: "Use CodeReady Workspaces to develop cloud-native applications from an OpenShift cluster."
 permalink: codewind-crw-quick-guide.html
-duration: 10 minutes
+duration: 15 minutes
 keywords: Codewind, CodeReady Workspaces, OpenShift
-objectives: ["Install CodeReady Workspaces and Codewind.", "Create a Codewind workspace within CodeReady Workspaces."]
+objectives: ["Install CodeReady Workspaces and Codewind.", "Create a Codewind workspace within CodeReady Workspaces.", Develop a simple microservice with the Codewind workspace."]
```

Comment:
Missing opening quotes before `Develop a simple microservice...` so change to:  `, "Develop...`